b.go

```diff
@@ -0,0 +1,4 @@
+// hoge
```

Comment:
__[Golint]__

package comment should be of the form "Package a ..."


<!-- Comment by [SideCI](https://sideci.com) -->